src/main.rs

```diff
@@ -35,7 +35,10 @@ fn main() {
 
     if your_color == Color::White {
         let m = match generate_move(&game, Color::White) {
-            Some(m) => m,
+            Some(m) => {
+                println!("{}", m);
```

Comment:
do we need this println here?

Reply:
Needed for both the prof's script and my script, I think there is also one more requirement for the prof's script to do later